app/src/main/java/com/babylon/wallet/android/data/gateway/model/TokenPriceResponse.kt

```diff
@@ -0,0 +1,32 @@
+package com.babylon.wallet.android.data.gateway.model
+
+import com.babylon.wallet.android.data.repository.cache.database.TokenPriceEntity
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import rdx.works.core.InstantGenerator
+
+@Serializable
```

Comment:
please add @SerialName for every network model property, it can be renamed and lead to issues after proguard/r8 is run

Reply:
>it can be renamed and lead to issues after proguard/r8 is run

that's not true because we have set up proguard rules for kotlinx serilization that do exactly this job! 😉 

However, indeed it "feels safer" to have the serial names there so I will add them